anda/system/uutils-coreutils-replace/uutils-coreutils-replace.spec

```
@@ -0,0 +1,120 @@
+%global coreutils_ver 9.3
+%if 0%{?fedora} >= 42
+### Temporary solution, will be fixed on newer Oniguruma releases.
+%global build_cflags %{__build_flags_lang_c} %{?_distro_extra_cflags} -std=c18 -std=gnu18
+%endif
+
+Name:           uutils-coreutils-replace
+Version:        0.0.29
+Release:        2%?dist
+Summary:        Cross-platform Rust rewrite of the GNU coreutils
+License:        MIT
+URL:            https://github.com/uutils/coreutils
+Source0:        %url/archive/refs/tags/%version.tar.gz
+Source1:        coreutils-colorls.sh
+Source2:        coreutils-colorls.csh
+Source3:        https://raw.githubusercontent.com/coreutils/coreutils/refs/heads/master/src/dircolors.hin
+Patch0:         coreutils-fix-metadata.diff
+Patch1:         coreutils-fix-seq-neg-num-tests.diff
+Patch3:         https://src.fedoraproject.org/rpms/coreutils/raw/rawhide/f/coreutils-8.32-DIR_COLORS.patch
+BuildRequires:  cargo
+BuildRequires:  clang-devel
+BuildRequires:  gcc-c++
+BuildRequires:  libselinux-devel
+BuildRequires:  make
+BuildRequires:  rustfmt
+BuildRequires:  selinux-policy-devel
+Requires:       glibc
+Provides:       coreutils
+Provides:       coreutils-common
+Conflicts:      uutils-coreutils
+
+%description
+uutils coreutils is a cross-platform reimplementation of the GNU coreutils in Rust.
+While all programs have been implemented, some options might be missing or different
+behavior might be experienced.
+
+This package replaces the GNU coreutils commands.
+
+
+%prep
```

```suggestion
%prep
%cargo_prep_online
```
Use `%cargo_prep_online` to generate the cargo build config